ws/swagger.yml

```diff
@@ -0,0 +1,38 @@
+swagger: "2.0"
```

Comment:
Swagger doc is a nice touch, though it might not be as suitable for documenting websocket APIs. Let's keep it for now.